.github/workflows/open-source-unit-tests.yml

```diff
@@ -20,12 +20,10 @@ on:
       image_tag:
         description: 'Marqo image tag. Examples: "1.1.0", "test" "latest"'
         required: true
-  push:
-    branches: 
-      - mainline
   pull_request:
```

Comment:
just clarifying this runs unit tests every time there's a push to a mainline or releases branch, right? If so, this might be a good thing to add to the `marqo` repo as well.

Reply:
Do you mean the `pull_requests` trigger? It runs every time there is a push to a branch that is in PR directed at mainline/releases. This would be really useful to be added to Marqo too

Reply:
yep i meant the `pull_requests` trigger. Does it also run when the pull request itself is created, or only on pushes to it? Also doing a bit of research, the `pull_request_review` trigger may also be cool to think about. But that can be done another time.